pkg/kube/inject/webhook.go

```diff
@@ -480,7 +480,15 @@ func escapeJSONPointerValue(in string) string {
 // adds labels to the target spec, will not overwrite label's value if it already exists
 func addLabels(target map[string]string, added map[string]string) []rfc6902PatchOperation {
 	patches := []rfc6902PatchOperation{}
-	for key, value := range added {
+
+	var addedKeys []string
+	for key := range added {
```

Comment:
You added this a hygiene, or to fix something specific ?

Reply:
reason: the tests (including some of the golden file matching, iirc) want a consistent ordering of patches. ordering it here (which in most cases is a list of one or two) makes it easy to verify behavior.

the alternative is to rewrite the testing logic to tolerate random ordering -- which is a lot of work to save a small amount of overhead in the injector logic.